backend/docs/api.md

```diff
@@ -0,0 +1,151 @@
+# API
+
+## Routes
+
+### GET /shop/:shop/
```

Comment:
is the trailing `/` necessary? There are later cases where it doesn't look like you need one. Should it be / does it need to be consistent?

Reply:
Yes, only the `POST` should have a trailing `/`. I've updated the paths.